extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_WikiaOrg_index.php

```diff
@@ -0,0 +1,32 @@
+<footer class="wds-global-footer-wikia-org">
+	<?php if ( isset ( $model['header'] ) ) : ?>
```

Comment:
Is this check necessary? The wikiaOrg model always has a ready, right? https://github.com/Wikia/app/pull/16924/files#diff-d9b21a7ab79a35e43e2166790df9efdcR229